spec/helper/revenuecat_internal_helper_spec.rb

```diff
@@ -127,6 +127,27 @@
     end
   end
 
+  describe '.write_changelog' do
+    let(:prepopulated_changelog) { 'mock prepopulated changelog' }
+    let(:changelog_latest_path) { './mock-path/CHANGELOG.latest.md' }
+
+    before(:each) do
+      allow(File).to receive(:write).with(changelog_latest_path, prepopulated_changelog)
```

Comment:
We could let it actually run the change in a tmp file and check that the file was correctly updated. I've done that in a few tests for extra security... But I think it's ok like this as well.

Reply:
I agree, we would have caught the issue with that test

Reply:
Updated in [d729229](https://github.com/RevenueCat/fastlane-plugin-revenuecat_internal/pull/28/commits/d729229b02a735bff288aa369e4db39a7e5d1ac3)